invenio_pages/services/results.py

```diff
@@ -75,6 +70,23 @@ def _results(self):
             else self._page_results
         )
 
+    def to_dict(self):
```

Comment:
moderate: why can't we use the parent class function? If I understand correctly the only difference is that we force disallow aggregations and fields' expansion, what was the reason behind this decision? Otherwise, I think we could live with the parent one.

Reply:
This was a weird issue found by @anikachurilova by which the sortBy params of the parent to_dict() were breaking the search.